src/io/flutter/run/common/CommonTestConfigUtils.java

```diff
@@ -64,11 +64,11 @@ public TestType asTestCall(@NotNull PsiElement element) {
     return null;
   }
 
-  Map<DartCallExpression, TestType> cachedCallToTestType;
+  private final Map<String, Map<DartCallExpression, TestType>> cachedCallToTestType = new HashMap<>();
```

Comment:
caching DartCallExpressions will cause us to hang on to old IntelliJ ASTs and leave us vulnerable to if IntelliJ recomputes AST nodes. It is safer to use file offsets as keys.

Reply:
Done.